xrmocap/transform/convention/keypoints_convention/__init__.py

```diff
@@ -15,8 +15,9 @@
 from . import campus, human_data, panoptic  # noqa:F401
 
 # yapf: enable
-KEYPOINTS_FACTORY['campus'] = campus.CAMPUS_KEYPOINTS
-KEYPOINTS_FACTORY['panoptic'] = panoptic.PANOPTIC_KEYPOINTS
+if isinstance(KEYPOINTS_FACTORY, dict):
+    KEYPOINTS_FACTORY['campus'] = campus.CAMPUS_KEYPOINTS
+    KEYPOINTS_FACTORY['panoptic'] = panoptic.PANOPTIC_KEYPOINTS
```

Comment:
What if it's not and when does it happen?

Reply:
When mmhuman3d is mocked, the imported `KEYPOINTS_FACTORY` will be taken as an `object` rather than a `dict`.
As the installation of mmhuman3d is fixed now, we can try to remove mock and install the package.

Reply:
It requires installing mmhuman3d without torch, otherwise we have to somehow let torch installed before mmhuman3d in `requirements/readthedocs.txt`.

Reply:
It will parse `.readthedocs.yml` for installation.

Reply:
After reading https://docs.readthedocs.io/en/stable/config-file/v2.html#id7, we can try the following file:
```yaml
version: 2

formats: all

python:
  version: 3.7
  install:
    - method: pip
      path: torch
    - requirements: requirements/docs.txt
    - requirements: requirements/readthedocs.txt
```